internal/bgp/bgpnative/native.go

```diff
@@ -0,0 +1,702 @@
+package bgpnative
```

Comment:
I had the same opinion here but decided to keep it for myself, but since Johannes expressed the same view I am now raising in support of just "native".

Reply:
I had `native` originally and someone else suggested `bgpnative`. Thats the only reason. Happy to change.

Reply:
I have changed backed to native. However, to resolve @fedepaol 's point. I did 

```	bgpfrr "go.universe.tf/metallb/internal/bgp/frr"
	bgpnative "go.universe.tf/metallb/internal/bgp/native"```